docs/en/navigation.yml

```
@@ -10,29 +10,29 @@ nav:
           - Scenarios: dce/scenario.md
           - Glossary: dce/terms.md
       - Installation:
-          - Briefs: install/intro.md
-          - Install dependencies: install/install-tools.md
-          - Install Community Release:
-              - Resource planning: install/community/resources.md
-              - Online install with kind: install/community/kind/online.md
-              - Online with K8s cluster: install/community/k8s/online.md
-              - Offline install with K8s cluster: install/community/k8s/offline.md
-          - Install Commercial Release:
-              - Deployment architecture: install/commercial/deploy-arch.md
-              - Deployment requirements: install/commercial/deploy-requirements.md
+          - Overview: install/intro.md
+          - Install Dependencies: install/install-tools.md
+          - Install Community Package:
```

```suggestion
          - Install Community Package:
```
Community Release 有 70 个额，这要全局替换吧